x-pack/test/functional/apps/ml/anomaly_detection/anomaly_explorer.ts

```diff
@@ -125,6 +126,12 @@ export default function ({ getService }: FtrProviderContext) {
         it('anomalies table is not empty', async () => {
           await ml.anomaliesTable.assertTableNotEmpty();
         });
+
+        it('should allow to attach anomaly swimlane embeddable to the dashboard', async () => {
```

Comment:
Maybe add a comment that this should be the last step in the anomaly explorer test as it leaves the ML area and the anomaly explorer is no longer available for further tests.

Reply:
Fixed in https://github.com/elastic/kibana/pull/68784/commits/c509d2f647dde130baf298d64fc3d3cb2eb61726